vars/rollingUpgradePipeline.groovy

```diff
@@ -96,8 +99,9 @@ def call(Map pipelineParams) {
                                 wrap([$class: 'BuildUser']) {
                                     dir('scylla-cluster-tests') {
                                         checkout scm
+                                        ArrayList base_versions_list = params.base_versions.contains('.') ? params.base_versions.split('\\,') : []
```

Comment:
1) why do you check that `base_versions` contain "." (dot) and then split by "," (comma)?
2) In Groovy split() returns empty Array if the string not found, I don't understand why do you need ternary if here...

Reply:
@yaronkaikov helped to understand why the `split()` didn't return an empty array when the string was empty, but that was the problem, so the "auto mode" wasn't working on "previous" version of this commit...

so, every version we have, OSS or enterprise always have a ".", so if it is not an empty string, the `split()` will work and return a singe or multiple element array, but if the string is emtpy (like it is expected in the "auto mode"), will not do the `split()`